frontend_tests/node_tests/compose.js

```diff
@@ -340,8 +340,8 @@ people.add(bob);
     range_length = selected_word.length;
     event.keyCode = undefined;
     event.which = 73;
-    event.metaKey = true;
-    event.ctrlKey = false;
+    event.metaKey = false;
+    event.ctrlKey = true;
```

Comment:
Is there any way we can tests here for mac environment, make sure it is only working for `cmd` not for `ctrl`(and similar in non-max env)?
 
Also at this point we should add one test, passing `true` for both cmd and ctrl key and compare no changes in compose box.

Reply:
@YJDave yeah, you just need tweak `set_global('navigator', {});` partway through the test to set it as Mac (and then later turn it off after that section of the test).

(In the Node tests, you can just edit things, e.g. `navigator.foo=bar` ... `navigator.foo = bar`).